docs/faq.rst

```diff
@@ -177,7 +177,7 @@ concrete vs. abstract values, you may want to read `Different kinds of JAX value
 Different kinds of JAX values
 ------------------------------
 
-In the process of transforming functions, JAX replaces some some function
+In the process of transforming functions, JAX replaces some function
```

Comment:
Ahaha this is the change someone made in google3 yesterday and I accidentally approved; I was supposed to make a matching PR so it wouldn't get dropped but having it brought in here through reverse-copybara is very convenient!